src/routes/accounts.py

```
@@ -1,3 +1,4 @@
+from types import NoneType
```

Ruff (0.11.9)

`types.NoneType` imported but unused

Remove unused import: `types.NoneType`

(F401)